website/blog/2024-10-05-kubernetes-blog.md

```diff
@@ -0,0 +1,157 @@
+    9. View the running front-end application.
+    ![running front-end application](img/building-a-kubernetes-application/running-application-locally.png)
+
+
+## Creating a pod
+You can use both the containers to create a pod. This way both the front-end and back end container applications can share resources, such as storage and network.
+
+**_Pod creation with existing containers_**
+
+1. Go to the **Containers** page.
+2. Select both the front-end and back-end containers.
+3. Click the **Create Pod** button.
+  ![create a pod from containers](img/building-a-kubernetes-application/creating-pod-from-containers.png)
+4. Click **Create Pod**.
+  ![copying containers to a pod](img/building-a-kubernetes-application/copying-containers-to-a-pod.png)
+5. View the newly created pod on the Pods page.
+6. Click the name of the pod and then click the **Summary** tab to view its summary.
+  ![viewing pod details](img/building-a-kubernetes-application/viewing-pod-details.png)
+
+
+**_Alternative: Pod creation with Kubernetes YAML_**
+
+You can generate a Kubernetes manifest for any existing pod or container and use it to create a local Kubernetes YAML file. Then, you can customize that file and create a pod from it.
+
+1. Go to the **Pods** page.
+2. Click the overflow menu icon corresponding to the pod.
+  ![overflow menu icon](img/building-a-kubernetes-application/overflow-menu-icon.png)
+3. Select the **Generate Kube** option from the dropdown list.
+4. View the Kubernetes YAML configuration in the **Kube** tab.
+  ![kube manifest](img/building-a-kubernetes-application/kube-manifest.png)
+5. Copy the configuration and paste it into a YAML file on your machine.
+6. Edit the YAML configuration and save it.
+7. Go to the **Pods** component page.
+8. Click **Play Kubernetes YAML**.
+  ![play kubernetes yaml](img/building-a-kubernetes-application/play-kubernetes-yaml.png)
+9. Select the YAML file from your machine.
+10. Click **Play** and then **Done**. 
+11. View the newly created pod on the same page.
+
+After creating the pod, set up a local Kubernetes cluster to deploy the pod.
+
+## Setting up a local Kubernetes cluster
+
+You can set up a local Kubernetes cluster. Once the cluster is connected and running, you can deploy your application on it. 
+
+Based on your preference, use the Kind or Minikube extension:
+1. [Install the extension](/docs/extensions/install) from the Extensions catalog.
+2. Create a Kubernetes cluster. See [Kind cluster creation](/docs/kind/installing-extension) or [Minikube cluster creation](/docs/minikube/installing-extension).
+
+Once a kubernetes cluster is created, you can view a running control plane node and a running Kubernetes service on the Kubernetes component page. The page also shows that the cluster is connected.
+
+## Deploying the application to Kubernetes
+
+You can deploy the pod or container to a Kubernetes cluster that has an active connection and and access it through a service. 
```

Comment:
```suggestion
You can deploy the pod to a Kubernetes cluster that has an active connection and and access it through a service. 
```

I would not say that we can deploy a container to Kubernetes, as it only possible through a Pod

Reply:
A quick question for understanding: If we cannot deploy a container to kubernetes, then why we get this option of `Deploy to Kubernetes` in the ACTIONS menu of the Containers page?

Reply:
In the Containers page, when choosing "Deploy to Kubernetes", a Pod is created (containing the container to deploy).

So effectively, if we consider the Podman Desktop experience, we can deploy a container to Kubernetes. I would change the sentence to something like below to clarify:

`You can deploy a pod or a container (inside a pod) to a Kubernetes cluster that ...`

Reply:
Thank you for the clarification, @feloy